.github/workflows/rust.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        rust_version: [1.65.0, stable]
+        rust_version: [1.66.0, stable]
     runs-on: ubuntu-20.04
 
     steps:
```

Comment:
add similar step as in this [commit](https://github.com/near/borsh-rs/pull/182/commits/f688da9bb8a44c749dae1ffe9baf2c714e37cfc3), so that master build doesn't fail or any new commits to this pr.
`time` and `time-macros` package have been updated.